cstrike/addons/amxmodx/scripting/ChatAdditions_Core.sma

```diff
@@ -1,4 +1,4 @@
-#include <amxmodx>
+
```

Comment:
```suggestion
#include <amxmodx>
```

Reply:
from amxmiscinc

#if !defined _amxmodx_included
#include amxmodx
#endif

my opinion: dodge it xD